packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -490,6 +490,7 @@ export class Repository extends Construct implements IRepository {
       const instances = props.documentDbInstanceCount ?? Repository.DEFAULT_NUM_DOCDB_INSTANCES;
       const dbCluster = new DatabaseCluster(this, 'DocumentDatabase', {
         masterUser: {username: 'DocDBUser'},
+        engineVersion: '3.6.0',
```

Comment:
Two things:

1.  This is good for the case where `props.database` is not supplied, but is it possible to add validation that the engine is correct for the code path where a caller specifies their own cluster and fail-fast?
1.  Can we add a test for this new behavior?

Reply:
It's a good idea. I'll see what I can cobble together.